examples/error_printer.cpp

```diff
@@ -118,7 +175,16 @@ int main()
 		}
 		catch (const toml::parse_error& err)
 		{
-			std::cout << err << "\n\n"sv;
+			std::cout << err;
+
+			auto line_num = err.source().begin.line;
+
+			if (auto line = toml::get_line(str, line_num))
+			{
+				std::cout << "\nLine "sv << line_num << ": "sv;
+				print(*line);
+			}
+			std::cout << "\n\n"sv;
```

Comment:
Ah, nice. Though, note you've only added this to the exception-based path - the exceptionless one lacks this feature. Ideally you'd split this new logic so that the get_line part would come after the exception branching, since it's the same in both cases.

Reply:
Just in case you missed it, I have addressed this comment of yours in the mean time, by adding a local helper function, `print_parse_error`, called by both the exception-based and the exceptionless path.

Reply:
Yep, I saw, looks good :)